### PR TITLE
Set up a "Hello world" project with two apps (Backend)

### PR DESCRIPTION
**Pull request by katarighe**
Link to the front-end part: [hello-rails-front-end](https://github.com/katarighe/hello-rails-front-end)

In this PR, I implement the following tasks and activities for the backend development project:

* [x] Create a new Rails API app called 'hello-rails-back-end'.
* [x] Initialize your project with Git.
* [x] Make sure that your project has a Postgres database set up.
* [x] Create a table for storing your messages and create 5 different greetings.
* [x] Create an API endpoint that selects a random greeting from your table (you will need a controller with an action and rails route)
* [x] In the README file for the Rails API, add a link to your React app.
* [x] When submitting for a review - use a link to the pull request in your Rails API.

### CODE REVIEW

**Review by OyePriscilla — COMMENTED** [review]
Hi @katarighe 👏, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Please kindly add a working link of the frontend version of your app to the README file and remove the live demo header if there is none for now. ℹ️ 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by julie-ify — COMMENTED** [review]
Hi @katarighe,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

* Kindly add the future features section on the readme file of the frontend repo.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app/models/message.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class Message < ApplicationRecord
```
Comment (ShadyShawkat) [inline app/models/message.rb:1]: - [ ] Kindly fix the following error that appears when all of the other package error are fixed. You can read more about cors and how to fix it [here](https://medium.com/@chris.lty07/configuring-for-cors-and-cookies-with-react-and-ruby-on-rails-1c30219cc2db).

image

Comment (ShadyShawkat) [inline app/models/message.rb:1]: - [ ] Please make sure that the app works properly. The following errors are shown when the app starts, to replicate this please clone the apps again and run them individually. These are mostly packages missing errors which should be easy to fix.

image

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @katarighe, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/message.rb (hunk 1 above)
Comment (katarighe) [reply app/models/message.rb:1]: ✅ It is now working fine.
image

Comment (katarighe) [reply app/models/message.rb:1]: ✅ It is now working fine for me right now.

Comment (katarighe) [reply app/models/message.rb:1]: ✅ Solved

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @katarighe    👍 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

image


Highlights
- [x] Good project organization ✔️
- [x] Nice use of Rails  features  ✔️  
- [x] Kudus on meeting the requirements as pointed the last time out by the last reviwer  👏 



Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**